WatchUI/WatchUI.py

```diff
@@ -56,12 +56,12 @@ class WatchUI:
     ROBOT_LIBRARY_VERSION = __version__
     ROBOT_LIBRARY_SCOPE = 'GLOBAL'
 
-    save_folder_path = "../Outputs"
+    save_folder_path = "./Outputs"
     starts_ssim = 1.0
     starts_format_image = "png"
-    path_to_tesseract_folder = r'C:\Program Files\Tesseract-OCR\tesseract.exe'
+    default_tesseract_path = ''
 
-    def __init__(self, outputs_folder: str = save_folder_path, ssim_basic: float = starts_ssim, format_image: str = starts_format_image, tesseract_path: str = path_to_tesseract_folder):
+    def __init__(self, outputs_folder: str = save_folder_path, ssim_basic: float = starts_ssim, format_image: str = starts_format_image, tesseract_path: str = default_tesseract_path):
```

Comment:
this is too long! how many characters is that? I will try to find some time and put a formatter in to the pipeline, but event for each team member I would strongly recommend to use black or yapf or something like this as IDE plugin and activate format on save, at least